install/InstallTools.py

```diff
@@ -898,12 +898,21 @@ def _data_container_dump(self, latest_id):
                 r.append(d)
             assert len(r) == 1000
             log_dir = Tools.text_replace("{DIR_VAR}/logs")
-            path = "%s/%s" % (log_dir, self.appname)
-            Tools.dir_ensure(path)
+            app_logs_path = "%s/%s" % (log_dir, self.appname)
+            Tools.dir_ensure(app_logs_path)
             path = "%s/%s/%s.msgpack" % (log_dir, self.appname, stopid)
             Tools.file_write(path, msgpack.dumps(r))
-        # now remove from redis
 
+            # rotate old msgpacks if we have logs > MAX_MSGPACKS_LOGS_COUNT (default: 50) file
+            # means 50k logs, we delete the oldest 10 files, 10k logs
+            msgpacks_count = MyEnv.config.get("MAX_MSGPACKS_LOGS_COUNT")
```

Comment:
should have a hardcoded sane default for that get

Reply:
done